includes/settings/tools.php

```diff
@@ -60,6 +62,25 @@ public function ajax_elementor_clear_cache() {
 		wp_send_json_success();
 	}
 
+	/**
+	 * Recreate kit.
+	 *
+	 * Recreate default kit (only when default kit does not exist).
+	 *
+	 * Fired by `wp_ajax_elementor_recreate_kit` action.
+	 *
+	 * @since 1.0.0
+	 * @access public
+	 */
+	public function ajax_elementor_recreate_kit() {
+		check_ajax_referer( 'elementor_recreate_kit', '_nonce' );
+
+		$id = Plugin::$instance->kits_manager->create_default();
```

Comment:
Check if there is an active kit because a user can click the "button" many times.

Reply:
Done.